code/part-two/client/README.md

```diff
@@ -0,0 +1,22 @@
+# Client
+
+## Development
+
+From this directory, 
+
+```bash
+npm install
+npm run build:dev
+```
+
+In another terminal instance,
+
+```bash
+npm run live-server
```

Comment:
I wasn't able to `npm run live-server` because "live-server: command not found". Though `npm install --save-dev live-server` fixed it. Maybe you have it globally installed?

In any case, you can also serve this app with docker by running `docker-compose up` from `code/part-two/`. It will be served at `localhost:8000`. The important distinction here, is serving it with docker/apache will proxy the REST API to `localhost:8000/api`, making it available to the client without CORS issues.

I think _probably_ the README should reflect the apache approach, but if you're going to list the live-server option, it should probably be in the dev dependencies.

Reply:
Ah yes, I forgot to include `live-server` in package.json! I was only using it to speed up my own development process while not working w/ the REST API as I didn't know how to configure docker/apache to do a live reload automatically. Makes me wonder now if there's some sort of livereload thing for webpack that I can enable when in a development environment? 🤔

I'm also voting for README to reflect Apache approach and will remove `live-server` from the README and npm scripts.